modules/core/test/image/testImageMorphology.cpp

```diff
@@ -68,10 +71,16 @@ TEST_CASE("Binary image morphology", "[image_morphology]")
 
       common_tools::imageDilatationRef(I_morpho_ref, connexity);
       vpImageMorphology::dilatation(I_morpho_tpl, (unsigned char)1, (unsigned char)0, connexity);
-      vpImageMorphology::dilatation(I_morpho, connexity);
+      vpImageMorphology::dilatation<unsigned char>(I_morpho, connexity);
 
       CHECK((I_morpho_ref == I_morpho_tpl));
       CHECK((I_morpho_ref == I_morpho));
+
+      #if defined(VISP_BUILD_DEPRECATED_FUNCTIONS)
+      vpImage<unsigned char> I_deprecated_dilatation = I;
+      vpImageMorphology::dilatation(I_deprecated_dilatation, connexity);
+      CHECK((I_morpho_ref == I_deprecated_dilatation));
+      #endif
```

Comment:
Testing deprecated functions should be removed

Reply:
Done in commit af0e02632. Thanks for the help =)